cache_test.go

```diff
@@ -294,6 +316,19 @@ func TestInt64Key(t *testing.T) {
 	if !bytes.Equal(val, []byte("abc")) {
 		t.Error("value not equal")
 	}
+	time.Sleep(2 * time.Second)
+	val, expiry, err := cache.GetIntWithExpiration(1)
+	if err != nil {
+		t.Error("err should be nil")
+	}
+	if !bytes.Equal(val, []byte("abc")) {
+		t.Error("value not equal")
+	}
+	now := time.Now()
+	if expiry != uint32(now.Unix()+1) {
+		t.Errorf("Expiry should be two seconds in the future but was %v", now)
```

Comment:
Do you mean one second in the future?

And the test can be more stable if we get the expiration before `time.Sleep`.

Reply:
Woops, good point.